server/drivers/futaba.c

```diff
@@ -307,17 +460,23 @@ futaba_init (Driver *drvthis)
 	/* Allocate and store private data */
 	p = (PrivateData *) calloc(1, sizeof(PrivateData));
 	if (p == NULL)
+	{
+		report(RPT_ERR, "%s: Private Data is NULL", drvthis->name);
```

Comment:
This message is somewhat misleading. We are simply out of memory. (Should never happen on a modern system due to overcommitting by the kernel.)